src/clangd-context.ts

```diff
@@ -50,9 +50,14 @@ export class ClangdContext implements vscode.Disposable {
     if (!clangdPath)
       return;
 
+    const rootPath: string = vscode.workspace.rootPath;
+    const targetCwd: string =
```

Comment:
nit: I guess we don't need three variables here, can we just inline as

```
  options: {cwd: vscode.workspace.rootPath || process.cwd() }
```

I don't think this is too cryptic to read
or just initialize directly as vscode.workspace.rootPath || process.cwd()

Reply:
done